### PR TITLE
Home: add a home page

### PR DESCRIPTION
**Pull request by nicordev**

### CODE REVIEW

File: templates/member/navConnection.twig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% if connected %}
```
Comment (olix21) [inline templates/member/navConnection.twig]: https://symfony.com/doc/current/security.html#fetch-the-user-in-a-template

File: templates/member/navConnection.twig [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <li class="nav-item">
+        <a class="nav-link" href="/member-profile">Mon profil</a>
```
Comment (olix21) [inline templates/member/navConnection.twig]: set the `href` to `#` when implementing a new template. Just to be sure you can search them later easily

File: templates/member/navConnection.twig (hunk 1 above)
Comment (nicordev) [reply templates/member/navConnection.twig]: Ok, I will change that with a member system branch.

File: templates/member/navConnection.twig (hunk 2 above)
Comment (nicordev) [reply templates/member/navConnection.twig]: Done.

**Pull request merged**